### ISSUE TITLE
Incorrect wording under Rules for Access control

### ISSUE THREAD

**Issue by PhilT** [dotnet/docs#13727]
"Accessibility for individual fields of a record type cannot is determined by" - doesn't make sense. As I'm learning I'm not quite sure what this is meant to say.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6b203042-7c41-709b-7db1-2157b1ba6d45
* Version Independent ID: caa4ea7d-d490-f66f-0a0c-b8f62e883f1b
* Content: [Access Control - F#](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/access-control#feedback)
* Content Source: [docs/fsharp/language-reference/access-control.md](https://github.com/dotnet/docs/blob/master/docs/fsharp/language-reference/access-control.md)
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**
